### ISSUE TITLE
OreExcavation detects it as only 1 time

### ISSUE THREAD

**Issue by Xaikii** [codetaylor/dropt#16]
https://pastebin.com/VRGx9NzS

Title says it all, I am using OreExcavation at leaves and when breaking all at once I only get 1 single item doesn't matter how I change the values

**Comment by codetaylor** [comment]
Ok, so here's what's happening.

The meta values for block states *aren't always the same as meta values for the block's item*.

For example, when I break an oak leaf block, it's meta value is `0`, but when broken, it changes the block state of all leaves around it to signal a leaf decay check, which changes the meta value of all leaves around it to `8`. This new value no longer matches.

Using Dropt's debug output shows that it matches a meta of `0` first, then tries to match a meta of `8` for all remaining leaves, which fails.

You can set a rule's debug output flag like this:

```json
{
  "rules": [
    {
      "debug": true,
      "match": {
        ...
      },
      ...
    }
  ]
}
```

Also, see [DEBUG.md](https://github.com/codetaylor/dropt/blob/master/DEBUG.md)

You can see the block state change here:

```
[EventHandler:onHarvestDropsEvent:69]: minecraft:leaves[check_decay=false,decayable=true,variant=oak]
[EventHandler:onHarvestDropsEvent:69]: minecraft:leaves[check_decay=true,decayable=true,variant=oak]
[EventHandler:onHarvestDropsEvent:69]: minecraft:leaves[check_decay=true,decayable=true,variant=oak]
[EventHandler:onHarvestDropsEvent:69]: minecraft:leaves[check_decay=true,decayable=true,variant=oak]
[EventHandler:onHarvestDropsEvent:69]: minecraft:leaves[check_decay=true,decayable=true,variant=oak]
```

If I use the following, it will only match the first block broken:

```json
"match": {
  "blocks": {
    "blocks": [
      "minecraft:leaves:0"
    ]
  }
}
```

However, if I use the following, it will match both states of the block:

```json
"match": {
  "blocks": {
    "blocks": [
      "minecraft:leaves:0,8"
    ]
  }
}
```

Unfortunately, there is nothing that I can do to change this behavior.

**Comment by codetaylor** [comment]
To assist in determining the meta values of blocks broken, I could introduce a new command:

```
/dropt verbose
```

When this mode is active, Dropt would log the `domain:path:meta` value of all blocks broken to the console. Do you think this would be helpful?

The output would look like this:

```
minecraft:leaves:0
minecraft:leaves:8
minecraft:leaves:8
minecraft:leaves:8
minecraft:leaves:8
minecraft:leaves:8
minecraft:leaves:8
minecraft:leaves:8
minecraft:leaves:8
minecraft:leaves:8
...
```

**Comment by Xaikii** [comment]
Yes this would definetly help.
Thanks

**Comment by codetaylor** [comment]
Feature available in version 1.12.2-1.8.0
